backend/open_webui/utils/middleware.py

```
@@ -1,21 +1,21 @@
 import time
 import logging
 import sys
+import requests
+import re
```

Ruff (0.12.2)

`re` imported but unused

Remove unused import: `re`

(F401)